### PR TITLE
Multi-language docs examples for Manually Committing Offsets examples

### PR DESCRIPTION
**Pull request by wetted**

### CODE REVIEW

File: test-suite/src/test/java/io/micronaut/kafka/docs/consumer/offsets/ack/ProductListener.java [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
+import io.micronaut.kafka.docs.Product;
 import io.micronaut.messaging.Acknowledgement;
 
 import static io.micronaut.configuration.kafka.annotation.OffsetReset.EARLIEST;
 import static io.micronaut.configuration.kafka.annotation.OffsetStrategy.DISABLED;
 
-class ProductListener {
+@Requires(property = "spec.name", value = "AckProductListenerTest")
+// tag::clazz[]
+@KafkaListener(offsetReset = EARLIEST, offsetStrategy = DISABLED) // <1>
```
Comment (sdelamo) [inline test-suite/src/test/java/io/micronaut/kafka/docs/consumer/offsets/ack/ProductListener.java:14]: why did you move the `@KafkaListener` to the class level ?

Comment (guillermocalvo) [reply test-suite/src/test/java/io/micronaut/kafka/docs/consumer/offsets/ack/ProductListener.java:14]: @sdelamo 

> why did you move the @KafkaListener to the class level ?

We need to show that line in the example, because it's linked to bullet point `<1> Committing offsets automatically is disabled`. So it does make sense to me.

**Review by guillermocalvo — APPROVED** [review]
